src/backend/src/node-rs-crc32-shim.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Shim for @node-rs/crc32 written in pure JavaScript.
+ * @module
+ */
+
+/** @type {undefined | Uint32Array} */
+let crcTable
+
+/**
+ * @param {string | Buffer} input
+ * @param {number | undefined | null} [initialState]
+ * @returns {number}
+ */
+function crc32(input, initialState = 0) {
```

Comment:
did you reference this from somewhere? If so, can you add the link?

Reply:
Done in 1b448451d5a4d23a4ecba7d578f02e6cc8ca45f4.